.gitignore

```diff
@@ -8,3 +8,6 @@ __pycache__
 # CDK asset staging directory
 .cdk.staging
 cdk.out
+
+# Config file
+config.yaml
```

Comment:
was wondering if this should be ignored at all - I know we talked about this, but having the exact params used for building would be a good thing to have committed.

Reply:
I looked at the config.yml and nothing in it is going to create a new CFN stack(which is why I wanted the yml to be committed to git) so I think we should be ok excluding it